cluster/ci/config/mirroring/codeready-toolchain/member_v0_1_quay

```diff
@@ -0,0 +1 @@
+registry.svc.ci.openshift.org/codeready-toolchain/member-operator-v0.1:member-operator quay.io/codeready-toolchain/member-operator:v0.1
```

Comment:
Do these *need* to be in two separate mapping files, handled by two separate jobs?

Reply:
I can move under one file, and let one cronjob handle it

Reply:
@petr-muller Addressed in bebeb2f